models/index.js

```diff
@@ -0,0 +1,9 @@
+const db = require('../db/connection');
+
+const readCategories = () => {
+    return db.query(`SELECT slug, description FROM categories;`).then((result) => result.rows);
```

Comment:
can you refactor this so that if the categories were patched to include more information this would be included in your query